File: app/src/main/java/com/zillennium/utswap/screens/setting/kycScreen/idTypeScreen/PassportFragment.java
package com.zillennium.utswap.screens.setting.kycScreen.idTypeScreen;

import android.content.Intent;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.RelativeLayout;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;

import com.zillennium.utswap.R;
import com.zillennium.utswap.databinding.FragmentKycPassportBinding;
import com.zillennium.utswap.screens.setting.kycScreen.CameraActivity;

public class PassportFragment extends Fragment {
    FragmentKycPassportBinding binding;

    public View onCreateView(@NonNull LayoutInflater inflater,
                             ViewGroup container, Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        binding = FragmentKycPassportBinding.inflate(getLayoutInflater());
        View root = binding.getRoot();

        RelativeLayout btnPassport = binding.btnPassport;
        btnPassport.setOnClickListener(view -> {
            Intent intent = new Intent(getContext(), CameraActivity.class);
            intent.putExtra("sample", R.drawable.ic_passport_front);
            startActivity(intent);
        });
        

        return root;
    }

    @Override
    public void onResume() {
        super.onResume();
        binding.getRoot().requestLayout();
    }



}

File: app/src/main/java/com/zillennium/utswap/screens/kyc/idTypeScreen/fragment/passport/PassportFragment.kt
package com.zillennium.utswap.screens.kyc.idTypeScreen.fragment.passport

import android.content.Intent
import com.zillennium.utswap.R
import com.zillennium.utswap.UTSwapApp
import com.zillennium.utswap.bases.mvp.BaseMvpFragment
import com.zillennium.utswap.databinding.FragmentKycPassportBinding
import com.zillennium.utswap.screens.kyc.idTypeScreen.camera.idCardCameraActivity.IDCardCameraActivity
import java.io.IOException

class PassportFragment :
    BaseMvpFragment<PassportView.View, PassportView.Presenter, FragmentKycPassportBinding>(),
    PassportView.View {

    override var mPresenter: PassportView.Presenter = PassportPresenter()
    override val layoutResource: Int = R.layout.fragment_kyc_passport

    override fun initView() {
        super.initView()
        try {
            binding.apply {

                btnPassport.setOnClickListener {
                    val intent = Intent(UTSwapApp.instance, IDCardCameraActivity::class.java)
                    intent.putExtra("sample", R.drawable.ic_passport_front)
                    startActivity(intent)
                }

            }
        } catch (error: Exception) {
            // Must be safe
        }
    }
}